frontend/src/modules/samples/actions.js

```diff
@@ -1,25 +1,25 @@
-import {createNetworkActionTypes, networkAction} from "../../utils/actions";
+import { createNetworkActionTypes, networkAction } from "../../utils/actions";
```

Comment:
The changes in this file just seem to be formatting changes. Are there any code changes?

Reply:
No, did you want me to revert these changes? I must've accidentally formatted. Trying my best not to format existing pages. :/

Reply:
If you can revert this file, sure. 👍